.gitignore

```diff
@@ -9,6 +9,7 @@ devel/
 /.vscode/*
 # except recommended extensions
 !/.vscode/extensions.json
+~/.vscode/settings.json
```

Comment:
This change can be reverted, due to its relation to linting, which is handled by #25 (the line can just be removed).

Reply:
I removed this line from the gitignore, sorry for the confusion

Reply:
I don't think I can resolve this one myself, but if I'm wrong let me know